.github/workflows/docbuild.yaml

```diff
@@ -0,0 +1,50 @@
+name: Documentation Build
+
+on:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: write
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v2
+        with:
+          path: matter
+          fetch-depth: 0
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: cache-pip
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-doc-pip
+      - name: Install base dependencies
+        working-directory: matter
+        run: |
+          sudo apt update
```

Comment:
Is it needed? You don't seem to install apt packages.

Reply:
It is not, thanks! It has now been removed.